pkg/rbd/rbd.go

```diff
@@ -0,0 +1,57 @@
+package rbd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// List retrieves and displays the Ceph block pools and their associated images.
+func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace string) {
+	blockPoolList, err := clientsets.Rook.CephV1().CephBlockPools(clusterNamespace).List(ctx, v1.ListOptions{})
+	if err != nil {
+		logging.Fatal(err)
+	}
+	// var blockPoolNames []string
+	var blockPoolNames map[string]string
+	blockPoolNames = make(map[string]string)
+
+	for _, blockPool := range blockPoolList.Items {
+		// blockPoolNames = append(blockPoolNames, blockPool.Name)
+		blockPoolNames[blockPool.Name] = "--"
+	}
+
+	cmd := "rbd"
+	for poolName, _ := range blockPoolNames {
+		args := []string{"ls", "--pool=" + poolName}
+		list, err := exec.RunCommandInOperatorPod(ctx, clientsets, cmd, args, clusterNamespace, clusterNamespace, true)
+		if err == nil && len(list) > 0 {
+			blockPoolNames[poolName] = strings.ReplaceAll(list, "\n", "")
+		}
+	}
+	PrintBlockPoolNames(blockPoolNames)
+}
+
+// PrintBlockPoolNames takes a map of block pool names to image names and prints them in a tabular format.
+func PrintBlockPoolNames(blockPoolNames map[string]string) {
+	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+
+	// Print header with column names
+	fmt.Fprintln(writer, "poolName\timageName\t")
+	fmt.Fprintln(writer, "--------\t---------\t")
```

Comment:
```suggestion
	fmt.Fprintln(writer, "poolName\timageName\t")
	fmt.Fprintln(writer, "--------\t---------\t")
```

can we use `logging` pkg for logging here and below

Reply:
done

Reply:
```
$ ./bin/kubectl-rook-ceph rbd ls
Info: poolName  imageName
Info: --------  ---------
Info: builtin-mgr       --
Info: replicapool       csi-vol-f090b7e1-a821-4a71-a7d2-7e3e2aaea8d1
```
@subhamkrai The output when I use `logging` is not clear; is it possible to use only `fmt.Fprintln` ?

Reply:
hmm can you prefix with `\n` for new line and see if that make sense? otherwise if not then feel free to use `fmt.Fprintln`